exercise-book/src/nrf52-tools.md

```diff
@@ -50,13 +55,13 @@ To access the USB devices as a non-root user, follow these steps:
     ATTRS{idVendor}=="1366", ENV{ID_MM_DEVICE_IGNORE}="1", TAG+="uaccess"
     ```
 
-3. Run the following command to make the new udev rules effective
+2. Run the following command to put the new udev rules into effect
 
     ```console
     sudo udevadm control --reload-rules
     ```
 
-4. (Optional) Disconnect and reconnect the dongle. Then check its permissions again.
+To check the permissions again, first disconnect and reconnect the dongle. Then run `lsusb`.
 
     ```console
```

Comment:
This block must now be outdented back to the left-hand margin. Outside of an itemised list, the vestigial four space indent forms a code block and that block is assumed to be Rust. And so you have ` ```console ` inside a code block, which is not valid Rust code.